src/rnmi.adoc

```diff
@@ -71,9 +71,14 @@ of holding.
 .Resumable NMI cause `mncause`.
 include::images/bytefield/mncause.edn[]
 
-The `mncause` CSR holds the reason for the NMI, with bit MXLEN-1 set to
-1, and the NMI cause encoded in the least-significant bits or zero if
-NMI causes are not supported.
+The `mncause` CSR holds the reason for the NMI.
+If the reason is an interrupt, bit MXLEN-1 is set to 1, and the NMI
+cause is encoded in the least-significant bits.
+If the reason is an interrupt and NMI causes are not supported, bit MXLEN-1 is
+set to 1, and zero is written to the least-significant bits.
+If the reason is an exception within M-mode that results in a double trap as
+specified in the Smdbltrp exception, bit MXLEN-1 is set to 0 and the
+least-significant bits are set to the exception cause code.
```

Comment:
The cause code for which exception?  It's probably sufficient here to just say that the lsb's are set to the exception cause code specified in the Smdbltrp extension.

Reply:
Yeah, referencing the Smdbltrp spec is probably best.  Using the "unexpected trap" term of art could confuse readers who haven't yet read the other spec.

Reply:
After writing it, I decided the answer is to do both: "unexpected trap, as defined in the Smdbltrp specification".

Reply:
I went back and forth, but I think I found a phrasing that is comprehensible to people who haven't read the other spec.